editor/patches/@emotion__react@11.1.2.patch

```diff
@@ -1,7 +1,7 @@
-diff --git a/node_modules/@emotion/react/dist/emotion-react.browser.cjs.js b/node_modules/@emotion/react/dist/emotion-react.browser.cjs.js
+diff --git a/dist/emotion-react.browser.cjs.js b/dist/emotion-react.browser.cjs.js
```

Comment:
All of the patches are file renames (to match the style used by pnpm's native patching support to make updating the patches later easier), and the removal of `/node_modules/<package_name>` from the file paths used in the diffs (like has happened here), so you can ignore the patch diffs